qiita_db/support_files/patches/63.sql

```diff
@@ -1,12 +0,0 @@
--- Nov 22, 2017
```

Comment:
Why was this patch removed and merged with 62.sql?

Reply:
We decided that we will only have on patch per dev cycle.

Reply:
Got it, 👍